modules/azure/aks/outputs.tf

```diff
@@ -8,3 +8,11 @@ output "kube_config" {
     cluster_ca_certificate = base64decode(azurerm_kubernetes_cluster.this.kube_admin_config[0].cluster_ca_certificate)
   }
 }
+
+output "azure_metrics_identity" {
```

Comment:
Are we calling this az_metrics or azure_metrics? I have a preference of azure_metrics but its not too important, just that we use the same name everywhere.

Reply:
I will change to azure instead.